CONTRIBUTORS.md

```diff
@@ -1602,8 +1604,6 @@
 
 -[@useramrit](https://github.com/useramrit)
 
--[@AlexFil1983] (https://github.com/AlexFil1983)
```

Comment:
Why did you remove this user?

Reply:
Check two or so lines below, it is a duplicate

Reply:
You're right, sorry. :)